### PR TITLE
New package: cinny-desktop-3.2.1

### PR DESCRIPTION
**Pull request by tranzystorekk**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/cargo-tauri/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+short_desc="Command line interface for building Tauri apps"
+maintainer="Marcin Puc <tranzystorek.io@protonmail.com>"
+license="Apache-2.0, MIT"
+homepage="https://tauri.app"
+changelog="https://raw.githubusercontent.com/tauri-apps/tauri/dev/tooling/cli/CHANGELOG.md"
+distfiles="https://github.com/tauri-apps/tauri/archive/refs/tags/tauri-cli-v${version}.tar.gz"
+checksum=4ac0985d750f7a4c6849ecd4bcdd26306b5dcf99c84f71762a038b290afbf781
+
+post_install() {
+	vlicense LICENSE_MIT
```
Comment (mhmdanas) [inline srcpkgs/cargo-tauri/template:18]: You can generate completions apparently: https://tauri.app/v1/api/cli/#completions

Comment (tranzystorek-io) [reply srcpkgs/cargo-tauri/template:18]: Yeah, but for cargo plugins they are effectively useless...

Comment (mhmdanas) [reply srcpkgs/cargo-tauri/template:18]: Fair enough.

**Comment by madoka773** [comment]
i want to know when it be merged

**Comment by tranzystorekk** [comment]
Closing as I'm no longer interested in having a separate desktop application for this.

Anyone is free to take it over though 😃

**Pull request closed without merging**